CHANGELOG.md

```diff
@@ -25,7 +33,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Contributors for this release
 
-- kirderf1, Caldw3ll, Dweblenod, Riotmode
+- kirderf1, Caldw3ll, Dweblenod, Riotmode, Carnie, Akisephila
```

Comment:
That update has been released already, so I recommend you merge 1.19 and then move the changelog and contributors to the new unreleased block at the top